### PR TITLE
istioctl rev+1 (making it buildable on linux)

### PR DESCRIPTION
**Pull request by zeph**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/linuxbrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [x] Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.

-----

### CODE REVIEW

File: Formula/istioctl.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Istioctl < Formula
   url "https://github.com/istio/istio.git",
       :tag      => "1.1.9",
       :revision => "879413c25b871ae6fd0df82bea1605baf12cba2e"
+  revision 1
```
Comment (iMichka) [inline Formula/istioctl.rb]: `revision 1 unless OS.mac?`

**Comment by zeph** [comment]
@iMichka fits?

**Comment by iMichka** [comment]
You need to rebase this on master.

**Comment by iMichka** [comment]
We are almost there :)

**Comment by iMichka** [comment]
Ping @zeph

**Comment by zeph** [comment]
ok, how about now? @iMichka

**Comment by iMichka** [comment]
Thanks @zeph !

**Comment by zeph** [comment]
:)

**Pull request closed without merging**